app/views/idv/in_person/address.html.erb

```diff
@@ -6,24 +6,10 @@
   <%= render PageHeadingComponent.new.with_content(t('in_person_proofing.headings.address')) %>
 <% end %>
 
-<% unless capture_secondary_id_enabled %>
-  <p>
-    <%= t('in_person_proofing.body.address.info') %>
-    <%= new_tab_link_to(
-          t('in_person_proofing.body.address.learn_more'),
-          MarketingSite.help_center_article_url(
-            category: 'verify-your-identity',
-            article: 'verify-your-identity-in-person',
-          ),
-        ) %>
-  </p>
-<% end %>
-
 <%= simple_form_for(
       form, url: url_for, method: 'PUT',
             html: { autocomplete: 'off' }
     ) do |f| %>
-    <% if capture_secondary_id_enabled %>
       <%= render ValidatedFieldComponent.new(
```

Comment:
I agree with Zach's comment below about unindenting and think everything nested in <% if capture_secondary_id_enabled %> should be unindented.